src/Resources/config/core.xml

```diff
@@ -51,23 +60,28 @@
             <argument type="service" id="sonata.admin.helper"/>
             <argument type="service" id="validator"/>
         </service>
+        <service id="Sonata\AdminBundle\Controller\HelperController" alias="sonata.admin.controller.admin"/>
```

Comment:
Same here, who is going to use this in autowiring?

Reply:
I have no idea for this one. I add it "just in case", but I can remove it.
My main goal was another service.

Reply:
Please remove that one.

Reply:
Done